### DISCUSSION TITLE
Adding a secondary access

### DISCUSSION THREAD

**Discussion by F0R93** [EddieDover/fvtt-player-achievements#88] [Ideas]
Hi there! I am a DM who happens to have just,,, SO much on my sidebar out of necessity for how I run my games. I'd love if there was a secondary access point to the achievements, such as having a lil button near the chatbar, or in the settings tab like in the Foundry Achievements module. This module has been working so well otherwise, I just hate having to disable other modules temporarily to be able to access the achievements

**Comment by EddieDover** [comment]
<strike>What about having a function call in the API to toggle the window, similar to what I did with fvtt-party-sheet? Then you could activate it with a macro and bind it to a key or whatever else you wanted?</strike>

Silly me, @F0R93 , this is already part of the API. You can just do this in a new macro set as a script type. Is that good enough or nah?

```
game["fvtt-player-achievements"].api.toggleAchievementWindow()
```

**Reply by F0R93** [reply]
That'll def work, but it'd be helpful to also have a secondary button, even if it's low on the list of priorities! Thank you for your speedy response!

**Reply by EddieDover** [reply]
> That'll def work, but it'd be helpful to also have a secondary button, even if it's low on the list of priorities! Thank you for your speedy response! 

Instead of that, I'll add an option that will completely relocate the button to an existing submenu, probably the player menu (top icon), that way it still works with modules that theme Foundry (and I don't have to get hacky with icon placement). 

I created an issue for this so keep an eye out there or join my Discord server. I'll let you try out beta builds to make sure it works for you before I push a release.

**Reply by EddieDover** [reply]
Just an FYI @F0R93 : In the last few updates, I added a button to the settings sidebar and in v1.8.3 I'm adding the ability to toggle the button placement from the primary menu column, into the Token Controls sub column.

I'll probably push the update this weekend, but I'm waiting to see what shakes out before then.

**Reply by EddieDover** [reply]
Going to close this discussion for now but if we need more tweaks or whatnot, feel free to keep discussing :D
